### ISSUE TITLE
Do not delete on focus but on start of typing

### ISSUE THREAD

**Issue by goranefbl** [mathiasbynens/jquery-placeholder#92]
Great script, thank you for this.

Because of auto focus on start, its important for me not to delete placeholder before starting with typing. Is it possible to rewrite it to delete placeholder on type instead focus ?

Thanks again,

great script.

**Comment by gudmundurmar** [comment]
Was @mlms13 fork ever merged into this library?

**Comment by gudmundurmar** [comment]
Was anything similar to @mlms13 fork added to this library? Why was it closed and not merged?

**Comment by mlms13** [comment]
@gudmundurmar I don't think it was ever merged. When they changed the default branch from `master` to `gh-pages` all of the PRs against the master branch were automatically closed.

The PR was open for about a year, but it was kind of a lot of work to keep it up-to-date with master, so eventually I stopped maintaining my fork. The PR is here if anyone else wants to tackle it: https://github.com/mathiasbynens/jquery-placeholder/pull/209

**Comment by Guillotine17** [comment]
+1

**Comment by valeriya9292** [comment]
+1
